sessions/recap-project-2/character-count-example/index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1, shrink-to-fit=no"
+    />
+    <title>Character Count Example</title>
+    <link rel="stylesheet" href="./css/styles.css" />
+  </head>
+  <body>
+    <form class="form">
+      <h1>Fix me</h1>
+      <label class="form__label" for="question">Question:</label>
```

Comment:
Using "Question" as form field is tiedly coupled to the quiz app. It makes this code more or less the exakt solution for the recap project assignment.

I think we can use something other so that students need to transfer this solution to the quiz-app.

For example: we can label the field  "Personal Message" (and change the `id` and `name` props accordingly)